src/Butil/Bit.Butil.Demo/Server/wwwroot/css/app.css

```
@@ -1,886 +1,2252 @@
+    background-color: transparent;
+}
+
+.td-cell {
+    display: flex;
+    overflow: hidden;
+    text-align: left;
+    line-height: 1.6;
+    padding: 0.75rem;
+    white-space: normal;
+    font-size: 0.75rem;
     word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)